src/Rules/PropertyConditions/ElementGroups.cs

```diff
@@ -19,6 +19,7 @@ static class ElementGroups
         // the following occurs for xaml expand/collapse controls
         private static Condition FocusableGroup = Group & IsKeyboardFocusable & (StringProperties.Framework.Is(Core.Enums.Framework.WPF) | StringProperties.Framework.Is(Core.Enums.Framework.XAML));
 
+        private static Condition WPF = Framework.Is(Core.Enums.Framework.WPF);
```

Comment:
Out of curiosity, what's the criterion for using a Create method or just putting these inline? Most of the Conditions use Create methods, so should we lean that way, even for simple ones? As an aside, we currently have 2 inline creators that are each more complex than the CreateWPRDataGridCellCondition method

Reply:
I'll change this. I made it a method because I originally thought it would be more complicated.